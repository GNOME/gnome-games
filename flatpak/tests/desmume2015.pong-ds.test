[Retro Reftest]
Path=/desmume2015/pong-ds
Core=/app/lib/libretro/desmume2015_libretro.so
Medias=PongDS_v1.1.0.nds;

[Options]
desmume_internal_resolution=256x192;512x384;768x576;1024x768;1280x960;1536x1152;1792x1344;2048x1536;2304x1728;2560x1920;
desmume_num_cores=1;2;3;4;
desmume_cpu_mode=jit;interpreter;
desmume_jit_block_size=12;11;10;9;8;7;6;5;4;3;2;1;0;100;99;98;97;96;95;94;93;92;91;90;89;88;87;86;85;84;83;82;81;80;79;78;77;76;75;74;73;72;71;70;69;68;67;66;65;64;63;62;61;60;59;58;57;56;55;54;53;52;51;50;49;48;47;46;45;44;43;42;41;40;39;38;37;36;35;34;33;32;31;30;29;28;27;26;25;24;23;22;21;20;19;18;17;16;15;14;13;
desmume_screens_layout=top/bottom;bottom/top;left/right;right/left;top only;bottom only;quick switch;hybrid/top;hybrid/bottom;
desmume_hybrid_layout_scale=1;3;
desmume_hybrid_showboth_screens=enabled;disabled;
desmume_hybrid_cursor_always_smallscreen=enabled;disabled;
desmume_pointer_mouse=enabled;disabled;
desmume_pointer_type=mouse;touch;
desmume_mouse_speed=1.0;1.5;2.0;0.125;0.25;0.5;
desmume_pointer_colour=white;black;red;blue;yellow;
desmume_pointer_device_l=none;emulated;absolute;pressed;
desmume_pointer_device_r=none;emulated;absolute;pressed;
desmume_pointer_device_deadzone=15;20;25;30;35;0;5;10;
desmume_pointer_device_acceleration_mod=0;1;2;3;4;5;6;7;8;9;10;11;12;13;14;15;16;17;18;19;20;21;22;23;24;25;26;27;28;29;30;31;32;33;34;35;36;37;38;39;40;41;42;43;44;45;46;47;48;49;50;51;52;53;54;55;56;57;58;59;60;61;62;63;64;65;66;67;68;69;70;71;72;73;74;75;76;77;78;79;80;81;82;83;84;85;86;87;88;89;90;91;92;93;94;95;96;97;98;99;100;
desmume_pointer_stylus_pressure=50;51;52;53;54;55;56;57;58;59;60;61;62;63;64;65;66;67;68;69;70;71;72;73;74;75;76;77;78;79;80;81;82;83;84;85;86;87;88;89;90;91;92;93;94;95;96;97;98;99;100;0;1;2;3;4;5;6;7;8;9;10;11;12;13;14;15;16;17;18;19;20;21;22;23;24;25;26;27;28;29;30;31;32;33;34;35;36;37;38;39;40;41;42;43;44;45;46;47;48;49;;
desmume_pointer_stylus_jitter=disabled;enabled;
desmume_load_to_memory=disabled;enabled;
desmume_advanced_timing=enabled;disabled;
desmume_firmware_language=Auto;English;Japanese;French;German;Italian;Spanish;
desmume_frameskip=0;1;2;3;4;5;6;7;8;9;
desmume_screens_gap=0;5;64;90;0;1;2;3;4;5;6;7;8;9;10;11;12;13;14;15;16;17;18;19;20;21;22;23;24;25;26;27;28;29;30;31;32;33;34;35;36;37;38;39;40;41;42;43;44;45;46;47;48;49;50;51;52;53;54;55;56;57;58;59;60;61;62;63;64;65;66;67;68;69;70;71;72;73;74;75;76;77;78;79;80;81;82;83;84;85;86;87;88;89;90;91;92;93;94;95;96;97;98;99;100;
desmume_gfx_edgemark=enabled;disabled;
desmume_gfx_linehack=enabled;disabled;
desmume_gfx_txthack=disabled;enabled;
desmume_mic_force_enable=disabled;enabled;
desmume_mic_mode=internal;sample;random;physical;

[Frame 2]
State=Refresh
Video=desmume2015.pong-ds.2.png
