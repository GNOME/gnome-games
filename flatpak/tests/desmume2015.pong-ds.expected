/desmume2015/pong-ds/Boot: OK
/desmume2015/pong-ds/Options: OK
/desmume2015/pong-ds/2/FastForward: OK
/desmume2015/pong-ds/2/Run: OK
/desmume2015/pong-ds/2/Video: OK
